php/libraries/BreadcrumbTrail.class.inc

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * This file represent a BreadcrumbTrail object
+ *
+ * PHP Version 7
+ *
+ * @category Breadcrumbs
+ * @package  User_Interface
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS;
+
+/**
+ * A BredcrumbTrail is a list of breadcrumbs to provide to the front-end
+ *
+ * @category Breadcrumbs
+ * @package  User_Interface
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+class BreadcrumbTrail
+{
+    /**
+     * A list of breadcrumbs.
+     *
+     * @var array<Breadcrumb>
```

Comment:
I'm not sure what the difference is, but I find `Breadcrumb[]` more readable. (Regardless, we don't have any standard for that right now and phan can extract either, so it's not important.)

Reply:
`array<Breadcrumb>` is more Java'ish 
I switched to `Breadcrumb[] `